routes/actor.js

```diff
@@ -0,0 +1,123 @@
+const { Router } = require('express');
+const router = Router();
+
+const Actor = require('../models/Actor');
+const init = require('../models/init');
+
+router.get('/init', (req, res) => {
+    Actor.find()
+        .exec()
+        .then((actors) => {
+            if (actors.length > 0) {
+                res.status(409).json({
+                    message: 'Actors database already exists',
+                });
+            } else {
+                Actor.insertMany(init.actors)
+                    .then((result) => {
+                        console.log(result);
+                        res.status(201).json({
+                            actors: result,
+                        });
+                    })
+                    .catch((error) => {
+                        console.log(error);
+                        res.status(500).json({
+                            error: error,
+                        });
+                    });
+            }
+        });
+});
+
+router.post('/add', (req, res) => {
+    const actor = new Actor({
+        name: req.body.name,
+    });
+    actor
+        .save()
+        .then((result) => {
+            console.log(result);
+            res.status(201).json({
+                addedActor: actor,
+            });
+        })
+        .catch((error) => {
+            console.log(error);
+            res.status(500).json({
+                error: error,
+            });
+        });
+});
+
+router.get('/list', (req, res) => {
```

Comment:
there is no need in word `/list` here. just empty route. 
you'd better to combine it with a `GET /:id` case. if there is no `id` provided the server has to return all actors